src/js/index/todolist.js

```diff
@@ -0,0 +1,121 @@
+const $todoInput = document.querySelector("#new-todo-title");
+const $todoList = document.querySelector("#todo-list");
+const $todoCount = document.querySelector("strong");
+const $filters = document.querySelector(".filters");
+
+$todoInput.addEventListener("keyup", onAddTodoItem);
+$todoList.addEventListener("click", onToggleTodoItem);
+$todoList.addEventListener("dblclick", onEditTodoItem);
+$filters.addEventListener("click", onFilter);
+
+function onAddTodoItem(event) {
+    const todoTitle = event.target.value;
+    const todoList = document.getElementById("todo-list");
```

Comment:
```suggestion
    const todoList = document.querySelector("#todo-list");
```

사용법이 유연한 querySelector 를 사용하는 것이 일반적입니다.  [1~4번 라인](https://github.com/woowacourse/js-todo-list-step1/pull/49/files#diff-619344f02ea8f876a6582bbe6d6eba4f94e73b7f256e62d8b232a261a978271bR4)과 일관성을 맞추는게 좋을거 같아요 :smile: